README.md

```diff
@@ -3,9 +3,12 @@
 # Ardite Core
 
 [![Build Status][1]][2]
+[![Clippy Linting Result][3]][4]
 
 [1]: https://travis-ci.org/ardite/ardite-core.svg?branch=master
```

Comment:
I'd rather keep them local. Its to avoid having 100+ links, remove a paragraph with two links and the needing to renumber everything.